extra/1-radio-stations.js

```diff
@@ -13,7 +13,13 @@
  * - Should return this array to use in other functions
  */
 
-// `getAllFrequencies` goes here
+ function getAllFrequencies(){
+   return listOfFrequencies;
+ };
+
+ const listOfFrequencies = [87, 87.5, 90, 92.5, 98.5, 100, 104.5, 108];
```

Comment:
This works, but for this function it would be better to define the variable inside the function like so:
```
 function getAllFrequencies(){
   const listOfFrequencies = [87, 87.5, 90, 92.5, 98.5, 100, 104.5, 108];
   return listOfFrequencies;
 };
```